inst/assets/book.scss

```diff
@@ -227,9 +228,148 @@ li.dropdown-header > span.menu-text {
 	  background: #393939;
 	}
 
+}
+
+/* --------------- Introductory workflow ---------------- */
+
+#workflow {
+
+	.content{
+	  margin-left: 4rem;
+	  margin-right: -4rem;
+	}
+
+	.quarto-other-links{
+	  font-size: 16px;
+	  right: 0.2rem;
+	  top: 0.1rem;
+	  margin-top: -3%;
+	}
+
+	.datatables{
+	  overflow-y: hidden;
+	}
+
+	strong {
+	  color : #007562
+	}
+
+	h1 {
+	  font-size: 36px;
+	  color: #070707;
+	  font-weight: 700;
+	  border-image: linear-gradient(to right, #007562, #007562) 1;
+	  border-bottom-style: solid;
+	  border-bottom-width: 4px;
+	  margin-bottom: 10%;
+	}
+	
+	h2 {
+	  margin-top: 3rem;
```

Comment:
Can you explain what this new format does? Why the default is not working here? I cannot see clear difference between default formatting and this.

Also, if this clearly improves the formatting, this could be implemented in whole book

Reply:
We wanted to create a style different from OMA to clearly distinguish between the two. There are indeed unnecessary elements, and I will remove them. What this formatting does is quite similar to OMA's, but with different colors and font formatting. The default formatting does not work here because all the page content is contained within a <div id="workflow"> tag, which applies the workflow formatting as a priority to the page.

Reply:
Can you also comment the lines so that it is easier to maintain?

Reply:
One more thing: Is it possible to specify that this formatting is applied only in this "introductory workflow". In the future, there might be other workflows, where we do not want to apply this formatting?

Have you rendered book (you can temporarily remove chapters from book.yml)? For me it does not look difference between "normal" chapters and workflows. (Looks good though)